plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -305,6 +325,18 @@ public void setHookPrefixes(List<String> hookPrefixes) {
         this.hookPrefixes.addAll(hookPrefixes);
     }
 
+    /**
+     * Sets JDK Home for tests
+     * @param testJDKHome JDK home to be used. {@code null} for using defaul system one.
+     */
+    public void setTestJDKHome(@CheckForNull File testJDKHome) {
```

Comment:
But you're not checking for null here? Would `@Nullable` make more sense?

Reply:
I use a stronger annotation so that the checks are enforced in usages within the methods. FindBugs does not enforce  checks for `@Nonnull`